src/modules/userSyncTest/dataPerProgramType.js

```diff
@@ -0,0 +1,206 @@
+                            includeAllAttributes: true,
+                            includeDeleted: true,
+                        })
+                    )
+                }
+
+                for (let j = 1; j <= _event; j++) {
+                    teiPromises.push(
+                        props.d2.Api.getApi().get('events', {
+                            page: `${j}`,
+                            pageSize: 50,
+                            orgUnit: `${orgUnit}`,
+                            ouMode: 'DESCENDANTS',
+                            includeAllAttributes: true,
+                            includeDeleted: true,
+                        })
+                    )
+                }
+            })
+
+            break
+        case global:
+            // if it's global or doesn't have specific settings
+            // with ou Parents and "DESCENDENT"
+
+            orgUnitList.orgUnitParents.forEach(orgUnit => {
+                for (let i = 1; i <= _tei; i++) {
+                    teiPromises.push(
+                        props.d2.Api.getApi().get('trackedEntityInstances', {
+                            page: `${i}`,
+                            pageSize: 50,
+                            ou: `${orgUnit}`,
+                            ouMode: 'DESCENDANTS',
+                            fields:
+                                'trackedEntityInstance,created,lastUpdated,orgUnit,trackedEntityType,coordinates,featureType,deleted,attributes[attribute,value,created,lastUpdated],relationships[trackedEntityInstanceA,trackedEntityInstanceB,relationship,relationshipName,relationshipType,created,lastUpdated,from[trackedEntityInstance[trackedEntityInstance],enrollment[enrollment],event[event]],to[trackedEntityInstance[trackedEntityInstance],enrollment[enrollment],event[event]],relative],enrollments[enrollment,created,lastUpdated,orgUnit,program,enrollmentDate,incidentDate,followup,status,deleted,trackedEntityInstance,coordinate,events[event,enrollment,created,lastUpdated,status,coordinate,program,programStage,orgUnit,eventDate,completedDate,deleted,dueDate,attributeOptionCombo,dataValues[dataElement,storedBy,value,created,lastUpdated,providedElsewhere]],notes[note,value,storedBy,storedDate]]',
```

Comment:
If these are always the same set of fields (or a subset of them are common) then I think they should be pulled out into a const variable

Reply:
(this is used 5 times in this file, and maybe other places as well)